lib/components/Header/HeaderMobile.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC, useState } from "react";
+import * as styles from "./styles";
+
+import {
+  LogoWithClaim,
+  ToggleNavButton,
+  NavMobile,
+} from "@klimadao/lib/components";
+import { urls } from "@klimadao/lib/constants";
+
+interface Props {
+  children: JSX.Element[];
```

Comment:
i dont think you need to type children if you use `FC`

Reply:
true, removed